newlib/libc/tinystdio/vfprintf.c

```diff
@@ -414,7 +414,18 @@ skip_to_arg(const CHAR *fmt_orig, my_va_list *ap, int target_argno)
             } else if (c == 'c') {
                 (void) va_arg (ap->ap, int);
             } else if (c == 's') {
+#ifdef VFPRINTF_S
```

Comment:
We don't need to include this check here -- this function is skipping args while finding positional parameters. The `%s` value will already be checked in the main `vfprintf_s` loop below.